nameko/cli/run.py

```diff
@@ -110,7 +110,13 @@ def _bad_call():
 def run(services, config, backdoor_port=None):
     service_runner = ServiceRunner(config)
     for service_cls in services:
-        service_runner.add_service(service_cls)
+        try:
```

Comment:
don't think this is the right place to catch these (`run` may be used from other libraries)

`nameko/cli/main.py` is probably the place (already catches and prints `CommandError`s)